src/plugins/data/common/field_formats/static.ts

```diff
@@ -24,7 +24,13 @@
 export { HTML_CONTEXT_TYPE, TEXT_CONTEXT_TYPE } from './content_types';
 export { FieldFormat } from './field_format';
 export { FieldFormatsRegistry } from './field_formats_registry';
-export { getHighlightRequest, asPrettyString, getHighlightHtml } from './utils';
+export {
+  getHighlightRequest,
```

Comment:
I removed this file and changed the way we export utils here https://github.com/elastic/kibana/pull/56975